workflows/eager.nf

```diff
@@ -136,6 +138,23 @@ workflow EAGER {
         if ( params.preprocessing_tool == 'fastp' && !adapterlist.extension.matches(".*(fa|fasta|fna|fas)") ) error "[nf-core/eager] ERROR: fastp adapter list requires a `.fasta` format and extension (or fa, fas, fna). Check input: --preprocessing_adapterlist ${params.preprocessing_adapterlist}"
     }
 
+    // QualiMap
+    if ( params.snpcapture_bed ) {
+      ch_snpcapture_bed_gunzip = Channel.fromPath(params.snpcapture_bed, checkIfExists: true).collect().map{
+        file ->
+            meta = file.simpleName
+        [meta,file]
+      }.branch{
+        meta, bed ->
+            forgunzip: bed[0].extension == "gz"
+            skip: true
+      }
+      ch_snpcapture_bed = GUNZIP_SNPBED(ch_snpcapture_bed_gunzip.forgunzip).gunzip.mix(ch_snpcapture_bed_gunzip).map{it[1]}
```

Comment:
```suggestion
      ch_snpcapture_bed = GUNZIP_SNPBED(ch_snpcapture_bed_gunzip.forgunzip).gunzip.mix(ch_snpcapture_bed_gunzip.skip).map{it[1]}
```
I find the style of running the module and assigning it's output all in one go to be hard to read and debug. Please reformat the code. Module should be ran, THEN the output assigned in separate Lines of Code.